service/client.ts

```diff
@@ -95,6 +95,7 @@ const client = {
   threads: new ThreadsPromiseClient(URL, null, opts),
   get,
   post,
+  authenticatedPost,
   put,
```

Comment:
I'm still feeling iffy by the fact that we're tagging onto this file... we should really use `service/index.ts` instead to surface these new methods so that we can get rid of this file once we're completely done with the migration.